packages/schema/docs/build/schema.md

```diff
@@ -145,6 +148,91 @@ Key | Required | Type | Description
+`required` | no | `boolean` | If this value is required or not.
+`placeholder` | no | `string` | An example value that is not saved.
+`default` | no | `string` | A default value that is saved the first time a Zap is created.
+`primary` | no | `boolean` | Use this field as part of the primary key for deduplication. You can set multiple fields as "primary", provided they are unique together. If no fields are set, Zapier will default to using the `id` field. `primary` only makes sense for `outputFields`; it will be ignored if set in `inputFields`. It only works in static `outputFields`; will not work in custom/dynamic `outputFields`. For more information, see [How deduplication works in Zapier](https://platform.zapier.com/build/deduplication).
+`dynamic` | no | [/RefResourceSchema](#refresourceschema) | A reference to a trigger that will power a dynamic dropdown.
+`search` | no | [/RefResourceSchema](#refresourceschema) | A reference to a search that will guide the user to add a search step to populate this field when creating a Zap.
+`choices` | no | [/FieldChoicesSchema](#fieldchoicesschema) | An object of machine keys and human values to populate a static dropdown.
+`list` | no | `boolean` | Acts differently when used in inputFields vs. when used in outputFields. In inputFields: Can a user provide multiples of this field? In outputFields: Does this field return an array of items of type `type`?
+`children` | no | `array`[[/FieldSchema](#fieldschema)] | An array of child fields that define the structure of a sub-object for this field. Usually used for line items.
+`dict` | no | `boolean` | Is this field a key/value input?
+`computed` | no | `boolean` | Is this field automatically populated (and hidden from the user)? Note: Only OAuth and Session Auth support fields with this key.
+`altersDynamicFields` | no | `boolean` | Does the value of this field affect the definitions of other fields in the set?
+`steadyState` | no | `boolean` | Prevents triggering on new output until all values for fields with this property remain unchanged for 2 polls. It can be used to, e.g., not trigger on a new contact until the contact has completed typing their name. NOTE that this only applies to the `outputFields` of polling triggers.
+`inputFormat` | no | `string` | Useful when you expect the input to be part of a longer string. Put "{{input}}" in place of the user's input (IE: "https://{{input}}.yourdomain.com").
+`meta` | no | [/FieldMetaSchema](#fieldmetaschema) | Allows for additional metadata to be stored on the field. Supports simple key-values only (no sub-objects or arrays).
+`isSafe` | no | `boolean` | Indicates if this authentication field is safe (not secret).
```

Comment:
I'm not sure about the name of this new property. Safe for what? Perhaps `isNoSecret` is more explicit? I know it violates some style guides because it is a negation, but I find it more clear.

Reply:
Good point, maybe `isConfidential` or `isSecret` convey the intent better without need for the negation?

Reply:
@Natay I don't believe that is an option, as we currently consider all auth fields secret, so the new flag needs to opt fields out of that.

Reply:
Ahh that makes sense, I'll have it as `isNotSecret` and add a note then, thanks for the heads up!

Reply:
@FokkeZB Fine to go with `isNoSecret`, but just wanted to point out that the description could do the work of explaining what `isSafe` means like what we currently have and still use it:

`Indicates if this authentication field is safe to e.g. be stored without encryption or displayed (not a secret).`